india_banking/overrides/payment_entry.py

```
@@ -120,42 +117,34 @@ def _get_reference_data(reference=None):
-		)
-	)
-	re_query_entry = re_query.run()
+	).run(as_dict=1)
 
-	existing_payment_entries = filters["existing_payment_entries"] or []
-	pe_query_entry = [d[0] for d in pe_query_entry] or []
-	re_query_entry = [d[0] for d in re_query_entry] or []
-
-	order_entry = pe_query_entry + re_query_entry
+	order_entry = [
+		(pe.payment_entry if pe.payment_entry_reference == 0 else pe.payment_entry)
```

Ruff (0.14.14)

[warning] Useless `if`-`else` condition

(RUF034)